test_rclcpp/test/test_services_server.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <test_rclcpp/srv/add_two_ints.hpp>
 
+#include <memory>
```

Comment:
The same question as https://github.com/ros2/demos/pull/80#discussion_r81456795 applies here.